Import error in `src/server.ts`

**Expected behavior**:
Code build successfully and runs

**Actual behavior**:
Error
```
#55 [server prod 4/4] RUN npm run build
#55 0.381 
#55 0.381 > polis@0.0.0 build
#55 0.381 > tsc
#55 0.381 
#55 15.01 src/server.ts(37,8): error TS2300: Duplicate identifier 'Config'.
#55 15.01 src/server.ts(44,17): error TS2307: Cannot find module './log' or its corresponding type declarations.
#55 15.01 src/server.ts(45,8): error TS2300: Duplicate identifier 'Config'.
#55 ERROR: process "/bin/sh -c npm run build" did not complete successfully: exit code: 2
------
 > [server prod 4/4] RUN npm run build:
0.381 
0.381 > polis@0.0.0 build
0.381 > tsc
0.381 
15.01 src/server.ts(37,8): error TS2300: Duplicate identifier 'Config'.
failed to solve: process "/bin/sh -c npm run build" did not complete successfully: exit code: 2
15.01 src/server.ts(44,17): error TS2307: Cannot find module './log' or its corresponding type declarations.
15.01 src/server.ts(45,8): error TS2300: Duplicate identifier 'Config'.
------
Error: Process completed with exit code 17.
0s
```

**To Reproduce**:
[Please read GitHub action output.](https://github.com/CivicTechTO/polis/actions/runs/11296884996/job/31422626386#step:5:1406)

**Additional context**:
After merging `dev` branch and accept deprecated changes.

--- a/server/src/server.ts
+++ b/server/src/server.ts
@@ -40,9 +40,6 @@ import CreateUser from "./auth/create-user";
 import Password from "./auth/password";
 import dbPgQuery from "./db/pg-query";
 
-// Re-import disassembled code to promise existing code will work
-import Log from "./log";
-import Config from "./config";
 import fail from "./utils/fail";
 
 import {

